base/Base.jl

```diff
@@ -529,6 +533,9 @@ function __init__()
     if haskey(ENV, "JULIA_MAX_NUM_PRECOMPILE_FILES")
         MAX_NUM_PRECOMPILE_FILES[] = parse(Int, ENV["JULIA_MAX_NUM_PRECOMPILE_FILES"])
     end
+    if get(ENV, "JULIA_USE_NEW_PARSER", "false") == "true"
```

Comment:
```suggestion
    if get_bool_env("JULIA_USE_NEW_PARSER", false) === true
```

Reply:
`get_bool_env` seems quite new . 
This is in our codebase

```julia
parse_from_env(::Type{Union{String,Nothing}}, name, default=""; is_src_nothing=isempty) =
    let val = get(ENV, name, default); is_src_nothing(val) ? nothing : val end
parse_from_env(::Type{String}, name, default="") = get(ENV, name, default)
parse_from_env(::Type{Vector{String}}, name, default="", dlm=",") =
    let v=get(ENV,name,default); split(v, dlm; keepempty=false) end
parse_from_env(::Type{Bool}, name, default="") = let v=get(ENV,name,default); lowercase(v) in ("1", "true") end
parse_from_env(::Type{Int}, name, default="") = let v=get(ENV,name,default); parse(Int, v) end
parse_from_env(::Type{URI}, name, default="") =
    let val = get(ENV, name, default) ;
        isempty(val) ?
            nothing :
            val |>                                                              # use norm_url ? # use URI
                val -> (startswith(val, r"\w+://") ? val : "https://$val") |>   # with scheme; alleviate RFC1808 from ENV
                val -> rstrip(val, '/') |> # without trailing slash                # <- URIs.normpath
                URI
    end
```

It better maps to parse and is more complete. Meanwhile your truthy/falsy map may be larger.